docs/contributing/triaging-github-issues/index.md

```diff
@@ -73,11 +77,37 @@ Check out [the docs on issue labeling for more info](/contributing/how-to-label-
 
 If an issue comes in as a question with a known answer it can be tempting to answer it and close the issue. However, the consequence of this approach is that the answer to a question others may have is now buried in a closed issue and may be hard to surface. The preferred solution is to get that answer documented in the main Gatsby documentation and connect the issue to an answer by including a docs link.
 
-When a question comes in and it is determined to have a known answer, find the existing documentation that explains it. If that documentation does not exist, is insufficient, or unclear, it is reasonable to provide an answer in a comment in the issue. However, the issue should remain open and be labled as documentation. If you are able, you can open a PR adding the provided answer to the official documentation.
+#### Bug Report
 
-### Resolution flowchart
+Bug Reports are issues that identify functionality in Gatsby that should work but does not in a given scenario. If an issue is a Bug Report it should include steps to reproduce the problem. If it doesn't, ask the issue filer for those steps and label the issue with `needs reproduction`.
+
+Attempt to reproduce the bug using the steps given. If that's not possible, ask for more information and label the issue as `needs more info`.
+
+If the reproduction is successful label the issue with `confirmed` and determine who is best suited to implement a fix. If it's approachable for the community consider the `help wanted` or `good first issue` labels. Otherwise, label with `inkteam to review` so it can be picked up by a Gatsby team member.
+
+![Flow chart for handling a Bug Report](./BugFlow.png)
+
+#### Feature Request
+
+Feature Requests are issues that request support for additional functionality not currently covered in the existing codebase. The first step in triaging a feature request is to determine if it's a reasonable request; this is a challenge and if you don't feel comfortable making this determination please label with `inkteam to review`. If it's clear that this isn't a feature it makes sense for Gatsby to implement, provide a comment explaining the decision making and close the issue.
+
+If it's determined to be a worthwhile feature the next decision point is whether the feature should be added to core or upstream. Upstream features should be labeled with `upstream` and include comments about the scope.
```

Comment:
Maybe we could add a blurb about the meaning of the word "upstream" in this context? I feel like plenty of devs use it (e.g. using the _name_ "upstream" in Git) without comprehension. Since we're asking people to make a decision, a little explanation might be helpful especially for folks newer to OSS. Could also be a good addition to the glossary!

Reply:
That's a good point and I admittedly had that moment when I was writing this and ignored it. I'll make some additions to clarify.